Pd 183 openshift shared volumes work with selinux (#487)
portworx/pxdocs: content/portworx-install-with-kubernetes/on-premise/openshift/_index.md

The following sections explain how to install Portworx with Kubernetes on OpenShift. There are two ways to
install Portworx on OpenShift: using a DaemonSet, and using the Portworx Operator component. For clusters running OpenShift 4 and above, we recommend you install Portworx using Operator.

> Please note that, if you're running Portworx on Openshift 3.11 or earlier, shared volumes work only if `SELinux` is disabled on the node or in Docker. This issue is caused by a bug in Kubernetes. The bug is fixed in Kubernetes 1.13.4, which is shipped with OpenShift 4.1.

Reviewer: Why not make this a note? 

```suggestion
{{<info>}}
**NOTE:** if you're running Portworx on Openshift 3.11 or earlier, shared volumes work only if `SELinux` is disabled on the node or in Docker. This issue is caused by a bug in Kubernetes, which is fixed in Kubernetes 1.13.4 and is shipped with OpenShift 4.1.
{{</info>}}
```

Author: @nathan-px Just that I didn't want to draw too much attention to the matter. After all, it's solved in recent versions. Anyway, let's make it a note.
